Dockerfile

```
@@ -0,0 +1,18 @@
+# Build: docker build -t concordium/ledger-app-builder .
+
+# Some setup for running might be required: https://developers.ledger.com/docs/nano-app/load/#2-load-the-application-from-inside-the-container-image
+# Run: docker run --rm -ti -v "$(realpath .):/app" --privileged concordium/ledger-app-builder
+
+# Run commands from makefile (load, delete, release, etc.)
+
+FROM ledgerhq/ledger-app-builder:latest
```

I could not build for Nano X. The issue is that this is the old image that they don't maintain anymore. Updating it to the correct one makes it build for both devices. You can see that our build pipeline also uses this image.
```suggestion
FROM ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
```

---

Nice, thanks! 😄